### DISCUSSION TITLE
ACE Bibliography

### DISCUSSION THREAD

**Discussion by lbrock** [Attempto/APE#37] [General]
Ran across the URL below the other day, and did some poking around in it:
Attempto Controlled English  -    https://typeset.io/topics/attempto-controlled-english-33v6rcnh
- It seems to be a serious bibliography of ACE-related papers, tho not all of the papers are about ACE specifically
- Navigation seems to have problems - sometimes get 404 errors when trying to go to pages on it, but then may work on next try.   The next page arrow seems to work the most reliably.
- At least some of the papers that indicate full-text actually do link to the actual paper.

**Comment by Kaljurand** [comment]
Thanks! This looks like an interesting website, with several interesting features (Copilot, x min summary, TL;DR, FAQ, Trace, explanations in multiple languages). However, there are many completeness and correctness errors on it, e.g.:

- searching for "Attempto Controlled English" and sorting by citation count shows many irrelevant papers at the top
- the affiliation of Norbert E. Fuchs is sometimes incorrectly shown as University of Tartu (e.g. in https://typeset.io/explore/papers/verbalizing-owl-in-attempto-controlled-english-1v2c6yfrr7)
- the various forms of summaries don't seem all that useful and are often incomplete (but I guess it's difficult to summarize 8-page research papers anyway better than what is already provided in the abstract)

So I'm not sure what would be the benefit compared to more traditional bibliographies, e.g.

- http://attempto.ifi.uzh.ch/site/pubs/
- https://scholar.google.com/scholar?q=%22Attempto+Controlled+English%22
